### DISCUSSION TITLE
When bit support vue？

### DISCUSSION THREAD

**Discussion by fuyunzhishang** [teambit/bit#4862] [FAQ] [answered]
When bit support vue？ Maybe we can publish a roadmap?

**Comment by itaymendel** [comment, marked as answer]
Hi, sadly we don't have an ETA on it yet. i will update here when we start building the integration.

**Reply by DG3D** [reply]
Hello ! 

Still no news on Vue support ?

**Reply by b5710546232** [reply]
+1

**Comment by xiaolee55** [comment]
so when we can use vue on bit harmony
